datadog/resource_datadog_dashboard.go

```diff
@@ -3978,6 +4041,7 @@ func getWidgetAxisSchema() map[string]*schema.Schema {
 		"include_zero": {
 			Type:     schema.TypeBool,
 			Optional: true,
+			Default:  false,
```

Comment:
Any reason to change this to have a default? I think the datadog-go-api client/api will omit if empty already?

Reply:
I want to say I had a reason, but I don't remember now (it's been a month) and I don't have my test setup anymore.

Reply:
Ahh ok. Yeah I think that the graph editor actually defaults to True here if its not specified. I think we should remove this line and let the field be omitted if unset due to this unless there is another reason 🙂

Reply:
Removed.